conf/nginx-alldomains.conf

```diff
@@ -26,9 +26,22 @@
 		index index.php;
 		alias /usr/local/lib/roundcubemail/;
 	}
-	location ~ /mail/config/.* {
-		# A ~-style location is needed to give this precedence over the next block.
-		return 403;
+	location ~ ^/mail/(config|temp|logs)/ {
+        	deny all;
+	}
+	location ~ ^/mail/(README|INSTALL|LICENSE|CHANGELOG|UPGRADING)$ {
+        	deny all;
+	}
+	location ~ ^/mail/(bin|SQL)/ {
+        	deny all;
+	}
+	location ~ ^/mail/(.+\.md)$ {
+        	deny all;
+	}
+	location ~ ^/mail/\. {
+        	deny all;
+        	access_log off;
```

Comment:
Based on recommendations from Roundcube [here](https://github.com/roundcube/roundcubemail/wiki/Installation#protect-your-installation) and [here](https://github.com/roundcube/roundcubemail/blob/master/.htaccess).